src/main/java/org/jvnet/hudson/test/recipes/WithPluginManager.java

```diff
@@ -90,7 +90,7 @@ public void setup(JenkinsRule jenkinsRule, WithPluginManager recipe) throws Exce
         public void decorateHome(JenkinsRule jenkinsRule, File home) throws Exception {
             Class<? extends PluginManager> c = recipe.value();
             Constructor<? extends PluginManager> ctr = c.getDeclaredConstructor(File.class);
-            jenkinsRule.setPluginManager(ctr.newInstance(home));
+            jenkinsRule.setPluginManager(ctr.newInstance(new File(home, "plugins")));
```

Comment:
To restore behavior for those tests requesting `UnitTestSupportingPluginManager`.